### PR TITLE
Standaridization of gke installation docs

### LINKED ISSUE #9096: [Docs] Standardizing Supported Platform Documentation - GKE
### Current State
Supported platform documentation for GKE is not in an organized order similar to EKS

### Desired State
Standardizing supported platform documentation of GKE Similar to EKS and AKS

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by aboobakersiddiqr63**
**Notes for Reviewers**

This PR fixes #9096 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ✓] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/installation/gke.md [hunk 1]
```diff
@@ -9,56 +9,94 @@ list: include
...
+## Access Meshery UI
+
+To access Meshery's UI, please refer to the [instruction](/tasks/accessing-meshery-ui) for detailed guidance.
+
+Note:   _You can also verify health of your system using [mesheryctl system check](/reference/mesheryctl/system/check)_
+
+# Suggested Reading
+
+<!-- {% include suggested-reading.html diffName="true" isDiffTag="true" diffTag=tag %} -->
+{% include related-discussions.html tag="mesheryctl" %}
```
Comment (leecalcote) [inline docs/pages/installation/gke.md:102]:
```suggestion
{% include suggested-reading.html language="en" %}

{% include related-discussions.html tag="meshery" %}
```

The logic in `suggested-reading.html` was _just_ updated, which means that it is now (hopefully) useful.

**Comment by l5io** [comment]
🚀 Preview for commit 95d8cc828076ca7e22f7c838fded065abc736cd6 at: https://65374a741878581aee12b91a--meshery-docs-preview.netlify.app

File: docs/pages/installation/gke.md (hunk 1 above)
Comment (aboobakersiddiqr63) [reply docs/pages/installation/gke.md:102]: yeah it is useful.

Comment (aboobakersiddiqr63) [reply docs/pages/installation/gke.md:102]: image

**Pull request merged**